### ISSUE TITLE
Adjust deploy action to take into account the branch the run was started on

### ISSUE THREAD

**Issue by adamgian** [adamgian/minify-selectors#55]
So that the code zipped in the release is from the correct branch.

**Comment by adamgian** [comment]
This should be it:
```
gh release edit ${{ github.event.inputs.tag }} --target=${{ github.ref_name }}
```

**Comment by adamgian** [comment]
Fixed in a feature branch commit 66b1c51dc1b2a16d37dea7f484c4aada20946f44
